src/data/plugins.json

```
@@ -244,6 +244,14 @@
           ],
           "badge": "community"
         },
+
+        {
+          "name": "cypress-geolocation-locale-faker",
+          "description": "A Cypress plugin to mock geolocation, timezone, language, and locale settings for end-to-end testing. Useful for simulating user environments worldwide.",
+          "link": "https://www.npmjs.com/package/cypress-geolocation-locale-faker",
+          "keywords": ["geolocation", "timezone", "locale", "language", "cypress", "plugin", "testing"],
+          "badge": "community"
+        },
```

```suggestion
        {
          "name": "cypress-geolocation-locale-faker",
          "description": "A Cypress plugin to mock geolocation, timezone, language, and locale settings for end-to-end testing. Useful for simulating user environments worldwide.",
          "link": "https://github.com/pradapjackie/cypress-geolocation-locale-faker",
          "keywords": ["geolocation", "timezone", "locale", "language", "plugin", "testing"],
          "badge": "community"
        },
```

All other links in this list are to the corresponding GitHub repo, not to the npm repository
The keyword "cypress" is not necessary in this list of Cypress plugins, as they all related to Cypress

You've chosen to add your plugin entry in the middle of the community list in the "Development Tools" section. There is no written rule about this, however new community plugins are usually added at the end of the community part of the corresponding section, not in the middle.

---

@MikeMcC399 Suggested changes are done from my end.